Resources/Views/base.leaf

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+        <meta charset="utf-8">
```

Comment:
The rest of the file is tab spaced, so this should be, too. (Or we should convert the whole template to spaces which I would prefer)

Reply:
Apologies! I went so far as to copy-paste the leading whitespace, but I guess GitHub's browser-based editor didn't respect that. I'm happy to amend to tabs or convert to spaces, or let you make the appropriate change yourself—totally up to you!

Reply:
Ah, that makes sense. If you're up for converting it to spaces, that would be greatly appreciated. Vapor style is typically 4 spaces for indent, so I'm not sure what happened here. Otherwise, just changing this one line to be a tab  is sufficient to merge.

Reply:
Alright, d047fee fixes base.leaf. 0905783 finishes conversion from tabs to spaces (including a weird mix in CSS). You're welcome to apply just d047fee.